kinto_http/session.py

```diff
@@ -69,12 +70,15 @@ def request(self, method, endpoint, data=None, permissions=None,
         while retry >= 0:
             resp = requests.request(method, actual_url, **kwargs)
             retry = retry - 1
+            headers = resp.headers
+            if hasattr(headers, "Alert"):
+                warnings.warn(str(headers['Alert']['message']), DeprecationWarning)
```

Comment:
If you look at how the header is produced: https://github.com/Kinto/kinto/blob/b0d34333f4aef8f0210838602f6b5aff1558056a/kinto/core/errors.py#L189-L193
You'll see that you need to deserialize the json first. I would show other keys as well. What about a string like `key1=value, key2=value2` ?

Also I wouldn't use the `DeprecationWarning` here. Or maybe you're right, alerts are supposed to be deprecated features of the server... Your call :)